### PR TITLE
feat: login layout

### PR DESCRIPTION
**Pull request by jeromeandrewong**
This PR is for #8 
SCR-20230910-svds

## Implementation
- CoreLayout
   - created a pseudo `Sidebar` component in `/CoreLayout`
   - added the `Sidebar` component to `CoreLayout.tsx`

- AuthCard
   - created `AuthCard` component which allows changing of the card title through `title` prop. (default value has been set to 'Login') 
   - allow overriding of card props using `forwardRef`

## Note
- input component left out
- button component left out
- sidebar content left for another issue

### CODE REVIEW

File: src/common/components/Auth/AuthCard.tsx
Comment (Aztriltus) [inline src/common/components/Auth/AuthCard.tsx:12]: Let's use heading tag here for semantic HTML. Since this is the only header, I think you can use h1. 

Also, if possible, allow changing this heading tag via prop. Maybe can create and use some polymorphic type

**Review by Aztriltus — CHANGES_REQUESTED** [review]
Could you also help to post a screenshot of new changes? I think it'll help with the review process! 

Left a small comment, otherwise code looks pretty good!

File: src/common/components/Auth/AuthCard.tsx
Comment (jeromeandrewong) [reply src/common/components/Auth/AuthCard.tsx]: Thanks for the review Zachary! I've implemented polymorphism to the component, however I am unsure if there are ways to improve my current implementation. For example typescript doesn't give errors when I attempt to add additional props in `AuthCard` component on `login.tsx`. For example if I added `as={Link}` without a `href` argument, typescript doesn't show me that it requires a `href` in my editor. It only errors out on page load.

**Review by Aztriltus — DISMISSED** [review]
LGTM! Thanks for the updates @jeromeandrewong 

The build error seems to be about env variables. Maybe they are not setup properly in netlify so I think that's outside the scope of this PR. @DavidLHW ?

**Comment by DavidLHW** [comment]
@Aztriltus @jeromeandrewong Build's failing due to outdated env vars on the branch -- this would be fixed in #4 , this PR would be blocked until that's merged

**Review by DavidLHW — APPROVED** [review]
CLGTM @ [edfefe8](https://github.com/AfterClass-io/afterclass.io-v2/pull/10/commits/edfefe88574885ca391995e0259eca5a7ae8c47e)

**Pull request merged**
